iceoryx_posh/source/runtime/posh_runtime_impl.cpp

```diff
@@ -114,6 +114,10 @@ PoshRuntimeImpl::getMiddlewarePublisher(const capro::ServiceDescription& service
             LogWarn() << "Service '" << service << "' already in use by another process.";
             errorHandler(Error::kPOSH__RUNTIME_PUBLISHER_PORT_NOT_UNIQUE, nullptr, iox::ErrorLevel::SEVERE);
             break;
+        case IpcMessageErrorType::SERVICE_DESCRIPTION_IS_FORBIDDEN:
+            LogWarn() << "Usage of service '" << service << "' is forbidden.";
```

Comment:
If we can propagate that it is due to being used internally it would probably be better.